src/applications/hca/actions.js

```diff
@@ -39,15 +54,17 @@ function callFakeSuccess(dispatch, status = HCA_ENROLLMENT_STATUSES.enrolled) {
     dispatch({
       type: FETCH_ENROLLMENT_STATUS_SUCCEEDED,
       data: {
-        applicationDate: '2018-01-24T00:00:00.000-06:00',
-        enrollmentDate: '2018-01-24T00:00:00.000-06:00',
+        applicationDate: '2019-04-24T00:00:00.000-06:00',
+        enrollmentDate: '2019-04-30T00:00:00.000-06:00',
         preferredFacility: '463 - CHEY6',
         parsedStatus: status,
+        effectiveDate: '2019-04-25T00:00:00.000-06:00',
```

Comment:
the `enrollment_status` endpoint now includes the `effectiveDate` of the current enrollment status